src/main/java/com/iota/iri/BundleValidator.java

```diff
@@ -142,6 +198,14 @@ public static boolean isInconsistent(List<TransactionViewModel> transactionViewM
         return (value != 0 || transactionViewModels.size() == 0);
     }
 
+    /**
+     * Traverses down the given {@code tail} trunk until all transactions that belong to the same bundle
+     * (identified by the bundle hash) are found and loaded.
+     *
+     * @param tangle connection to the persistence layer
+     * @param tail should be the tail transaction of the bundle
```

Comment:
before you called it `last` transaction - decide on one :)

Reply:
`last`